docs(controller): add v1.0.1 release notes (#7423)
influxdata/docs-v2: content/telegraf/controller/reference/release-notes.md

weight: 101
---

## v1.0.1 {date="2026-07-01"}

> <!-- Link only be on the latest version, update and move with new versions. -->

Reviewer: ```suggestion
<!-- Update and move the link to the latest version. -->
```
